### ISSUE TITLE
Update our CI/CD Docker images to use Qt 6.6.3

### ISSUE THREAD

**Issue by guihkx** [nuttyartist/notes#668]
Qt 6.6.3 has been [released today](https://www.qt.io/blog/qt-6.6.3-released), and it will be the last release of the 6.6 series.

This release also fixes #611, so this is probably a good time to update the Qt 6 version we ship with our packages.

cc @zjeffer

**Comment by nuttyartist** [comment]
Yes! I'm waiting for Qt 6.7, it should support variable font files (https://www.qt.io/blog/text-improvements-in-qt-6.7) which will probably reduce our binary size a good deal.
